.github/workflows/url-checker.yml

```diff
@@ -1,23 +1,41 @@
 name: URL Checker
 
 on:
-  workflow_dispatch:
+  pull_request:
+    paths:
+      - '5.0/**'
   push:
     branches:
       - master
+    paths:
+      - '5.0/**'
+  workflow_dispatch:
 
 jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 1

```

Comment:
Do we no longer need fetch depth?

Reply:
That's correct, `1` is the [default value](https://github.com/actions/checkout?tab=readme-ov-file#checkout-v4).